test/auro-input.test.js

```diff
@@ -212,6 +212,198 @@ describe('auro-input', () => {
 
     await expect(el).to.be.true;
   });
+
+  it('handles credit card formatting - starts with "4" is Visa', async () => {
+    const el = await fixture(html`
+      <auro-input id="format-ccWithIcon" format='{"creditCard": true, "icon": true}' icon="true" label="Credit Card Number with Icon" required></auro-input>
+    `);
+
+    await expect(el).to.be.accessible();
```

Comment:
I wouldn't check to.be.accessible on every test. These tests should focus on setting the right card icon.

Reply:
Good point - removing this.